diff -bur Net_LDAP2-2.0.0.orig/usr/share/pear/Net/LDAP2/Entry.php Net_LDAP2-2.0.0/usr/share/pear/Net/LDAP2/Entry.php
--- Net_LDAP2-2.0.0.orig/usr/share/pear/Net/LDAP2/Entry.php	2008-06-01 18:19:49.778779445 +0000
+++ Net_LDAP2-2.0.0/usr/share/pear/Net/LDAP2/Entry.php	2008-06-01 18:20:34.254286396 +0000
@@ -2,7 +2,7 @@
 /* vim: set expandtab tabstop=4 shiftwidth=4: */
 
 require_once 'PEAR.php';
-require_once 'Util.php';
+require_once 'Net/LDAP2/Util.php';
 
 /**
 * Object representation of a directory entry
Tylko w Net_LDAP2-2.0.0/usr/share/pear/Net/LDAP2: Entry.php~
diff -bur Net_LDAP2-2.0.0.orig/usr/share/pear/Net/LDAP2/Filter.php Net_LDAP2-2.0.0/usr/share/pear/Net/LDAP2/Filter.php
--- Net_LDAP2-2.0.0.orig/usr/share/pear/Net/LDAP2/Filter.php	2008-06-01 18:19:49.778779445 +0000
+++ Net_LDAP2-2.0.0/usr/share/pear/Net/LDAP2/Filter.php	2008-06-01 18:20:41.291296976 +0000
@@ -2,7 +2,7 @@
 /* vim: set expandtab tabstop=4 shiftwidth=4: */
 
 require_once 'PEAR.php';
-require_once 'Util.php';
+require_once 'Net/LDAP2/Util.php';
 
 /**
 * Object representation of a part of a LDAP filter.
Tylko w Net_LDAP2-2.0.0/usr/share/pear/Net/LDAP2: Filter.php~
diff -bur Net_LDAP2-2.0.0.orig/usr/share/pear/Net/LDAP2.php Net_LDAP2-2.0.0/usr/share/pear/Net/LDAP2.php
--- Net_LDAP2-2.0.0.orig/usr/share/pear/Net/LDAP2.php	2008-06-01 18:19:49.792113431 +0000
+++ Net_LDAP2-2.0.0/usr/share/pear/Net/LDAP2.php	2008-06-01 18:20:15.833386125 +0000
@@ -21,13 +21,13 @@
 * Package includes.
 */
 require_once 'PEAR.php';
-require_once 'LDAP2/RootDSE.php';
+require_once 'Net/LDAP2/RootDSE.php';
 require_once 'Net/LDAP2/Schema.php';
-require_once 'LDAP2/Entry.php';
-require_once 'LDAP2/Search.php';
-require_once 'LDAP2/Util.php';
-require_once 'LDAP2/Filter.php';
-require_once 'LDAP2/LDIF.php';
+require_once 'Net/LDAP2/Entry.php';
+require_once 'Net/LDAP2/Search.php';
+require_once 'Net/LDAP2/Util.php';
+require_once 'Net/LDAP2/Filter.php';
+require_once 'Net/LDAP2/LDIF.php';
 
 /**
 *  Error constants for errors that are not LDAP errors.
Tylko w Net_LDAP2-2.0.0/usr/share/pear/Net: LDAP2.php~
